modules/registry-removed.adoc

```diff
@@ -20,4 +20,4 @@
 
 On platforms that do not provide shareable object storage, the OpenShift Image Registry Operator bootstraps itself as `Removed`. This allows `openshift-installer` to complete installations on these platform types.
 
-After installation, you must edit the Image Registry Operator configuration to switch the `managementState` from `Removed` to `Managed`.
+After installation, you must edit the Image Registry Operator configuration to switch the `managementState` from `Removed` to `Managed`. Then, you must configure storage.
```

Comment:
```suggestion
After installation, you must edit the Image Registry Operator configuration to switch the `managementState` from `Removed` to `Managed`.  When this is completed, then you must configure storage.
```

Reply:
@stevsmit I don't think there is style guidance on this but I do believe it translates a little better. Your call man.